tex/gregoriotex-signs.tex

```diff
@@ -1630,24 +1642,22 @@
 \gre@count@shiftaftermora=5
 
 \def\gresetshiftaftermora#1{%
-  \IfStrEq{#1}{always}%
-    {\gre@count@shiftaftermora=5}%
-    {\IfStrEq{#1}{notesonly}%
+  \IfStrEqCase{#1}{%
+    {always}%
+      {\gre@count@shiftaftermora=5}%
+    {notesonly}%
       {\gre@count@shiftaftermora=4}%
-      {\IfStrEq{#1}{barsonly}%
-        {\gre@count@shiftaftermora=3}%
-        {\IfStrEq{#1}{barsnotextonly}%
-          {\gre@count@shiftaftermora=2}%
-          {\IfStrEq{#1}{barsinsideonly}%
-            {\gre@count@shiftaftermora=1}%
-            {\IfStrEq{#1}{never}%
-              {\gre@count@shiftaftermora=0}%
-              {\gre@error{Unrecognized option "#1" in gresetshiftaftermora , use 'always', 'barsonly', 'notesonly', 'barsnotextonly', 'barsinsideonly' or 'never'}}%
-            }%
-          }%
-        }%
-      }%
-    }%
+    {barsonly}%
+      {\gre@count@shiftaftermora=3}%
+    {barsnotextonly}%
+      {\gre@count@shiftaftermora=2}%
+    {barsinsideonly}%
+      {\gre@count@shiftaftermora=1}%
+    {never}%
+      {\gre@count@shiftaftermora=0}%
+    }[% all other cases
+      \gre@error{Unrecognized option "#1" in gresetshiftaftermora , use 'always', 'barsonly', 'notesonly', 'barsnotextonly', 'barsinsideonly' or 'never'}%
```

Comment:
Looks globally ok, thanks! Maybe the other error messages could also list all the possibilities, like this one does? This would make error messages more helpful